configs/zero123.yaml

```diff
@@ -0,0 +1,143 @@
+name: "zero123"
+tag: "${rmspace:${system.prompt_processor.prompt},_}"
```

Comment:
Consider replacing the tag with the data image path since there is no prompt for this exp, i.e. `${rmspace:${basename:${data.image_path}},_}"` but not checked. See `threestudio/utils/config.py` for the custom function.